### ISSUE TITLE
`<title>` tag incorrect conversion

### ISSUE THREAD

**Issue by guillheu** [lpil/html-lustre-converter#15]
When converting
```html
<head>
<title>A Title</title>
</head>
```

The resulting Gleam code :
```gleam
html.html(
  [],
  [html.head([], [html.title([], [html.text("A Title")])]), html.body([], [])],
)
```
... incorrectly uses the function `html.title`

Instead of `html.title([], [html.text("A Title")])` it should be `html.title([], "A Title")`

**Comment by lpil** [comment]
Thank you

**Comment by giacomocavalieri** [comment]
This was closed by #18!
